index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>collabortion project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p id="tribute-info">The man who saved a billion lives</p>
+        
+        <a href="#" id="tribute-link" target="_blank"></a>
+        <div id="img-div">
+            <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="" id="image">
+            <div id="img-caption">
+                Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+            </div>
+        </div>
+        
+    </main>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
this link is already present on line 8 and that is where it should be in the head section for HTML best practices I would suggest removing this code 💨  and paying close attention so these small errors can be avoided. 🥇